### DISCUSSION TITLE
How can I change font style from Custom CSS

### DISCUSSION THREAD

**Discussion by renraw** [Eugeny/tabby#6012] [Q&A]
Hi guys, simple question, how can I change font style for the terminal console from option on Settings --> Appearance --> Custom CSS
For example set italic

Tks.

**Comment by Eugeny** [comment]
Custom CSS does not affect terminal contents, and specifically making it italic is not possible.

**Comment by renraw** [comment]
Oh ok tks, so how can I change the boldness or make the letters look right of the terminal?, cause its looks blurry or aliased to me, I have already tried disabling gpu acceleration, windows dpi, scaling, (cant find hterm option) I love this terminal but even putty looks better :(
